sites/partners/src/listings/PaperListingForm/index.tsx

```diff
@@ -184,6 +184,9 @@ const ListingForm = ({ listing, editMode }: ListingFormProps) => {
     if (status) {
       formData = { ...formData, status }
     }
+    if (!Object.values(formData.neighborhoodAmenities).some((item) => item.trim() !== "")) {
```

Comment:
This is not intuitive, but we handle this kind of data validation in formatter classes we have added that live under `listings/PaperListingForm/formatters/`. The `AdditionalMetadataFormatter` file would be a good place to add this logic, we have had to do similar fixes to this there as well.